.github/workflows/general.yml

```diff
@@ -57,11 +58,11 @@ jobs:
       - name: Compile integration tests
         working-directory: ./daemon/integration-tests
         run: cargo test --no-run --locked
-      - if: ${{ matrix.os == 'ubuntu-latest' }}
+      - if: ${{ matrix.os == 'ubuntu-latest' && matrix.nvim-version == 'v0.10.1' }}
         name: Run integration tests
         working-directory: ./daemon/integration-tests
         run: cargo test
-      - if: ${{ matrix.os == 'macos-latest' }}
+      - if: ${{ matrix.os == 'macos-latest' || matrix.nvim-version == 'v0.9.5' }}
```

Comment:
We do now have two-by-two = four CI scenarios and *one* of them runs the whole fuzzer. That seems reasonable to me? :)